### PR TITLE
Review entity

### PR DESCRIPTION
**Pull request by Mabotse-Mosima**
final updated review

### CODE REVIEW

File: src/main/java/za/ac/cput/factory/ReviewFactory.java
Comment (natheerShade13) [inline src/main/java/za/ac/cput/factory/ReviewFactory.java:1]: This needs a method that builds a random ReviewID.

File: src/main/java/za/ac/cput/repository/IReviewRepository.java
Comment (natheerShade13) [inline src/main/java/za/ac/cput/repository/IReviewRepository.java:1]: This interface only needs the List<Review> getAll() method.

File: src/main/java/za/ac/cput/repository/IRepository.java
Comment (natheerShade13) [inline src/main/java/za/ac/cput/repository/IRepository.java:1]: These 4 methods need to be here.

File: src/main/java/za/ac/cput/factory/ReviewFactory.java
Comment (Mabotse-Mosima) [reply src/main/java/za/ac/cput/factory/ReviewFactory.java:1]: noted

**Pull request closed without merging**